src/Components/Alert/Hooks/__tests__/AlertProvider.jest.tsx

```diff
@@ -95,11 +94,11 @@ describe("AlertProvider", () => {
 
     await flushPromiseQueue()
 
-    // expect(screen.getByText("Add Filters:")).toBeInTheDocument()
     expect(screen.getByText("Andy Warhol")).toBeInTheDocument()
+    expect(screen.getByText("Add Filters")).toBeInTheDocument()
 
     // transition to filters step
-    // screen.getByTestId("addFilters").click()
+    screen.getByTestId("moreFilters").click()
```

Comment:
At this point I'd like to test the **More Filters ⟩** element, but I am struggling with the correct usage of `mockResolveOperation`

|Skeleton|Rendered|
|---|---|
|![skeleton](https://github.com/artsy/force/assets/140521/2f575bb1-0bf9-443b-be59-35dcfbb1ba62)|![rendered](https://github.com/artsy/force/assets/140521/5019d93a-dfd6-40bc-ab03-1b5bc38337bb)|

Reply:
Fixed in [a8a9d21](https://github.com/artsy/force/pull/13489/commits/a8a9d21062647944441a86b5a99dbca73b123f3e) h/t @gkartalis